mnist/README.md

```diff
@@ -47,7 +47,7 @@
 5. **parallelTrialCount**: The maximum number of hyperparameter sets that Katib should train in parallel. The default value is 3.
 6. **maxTrialCount**: The maximum number of trials to run.
 7. **maxFailedTrialCount**: The maximum number of failed trials before Katib should stop the experiment.
-8. **algorithm**: The search algorithm that you want Katib to use to find the best hyperparameters or neural architecture configuration. 
+8. **algorithm**: The search algorithm that you want Katib to use to find the best hyperparameters or neural architecture configuration. Search algorithms   available in Katib are Grid search, Random search, Bayesian optimization, Hyperband, Tree of Parzen Estimators (TPE), Covariance Matrix Adaptation Evolution Strategy (CMA-ES), Neural Architecture Search based on ENAS, Differentiable Architecture Search (DARTS).
```

Comment:
We need to give the exact value which user needs to enter in this field. If user enters 'Adaptation Evolution Strategy (CMA-ES)', that value won't work right ?

You can say following:
Search algorithm to find the best hyper parameters. Value must be one of following:
1.
2.
3.

Reply:
Done @deepio-oc . Currently darts is in alpha with limited support, we will not be able to see the results in kubeflow ui n case of darts, rest are added.